site/app/templates/admin/admin_gradeable/AdminGradeableDates.twig

```diff
@@ -34,14 +34,20 @@
     <input name="grade_released_date" id="date_released" class="date_picker" type="text" value="{{ gradeable.getGradeReleasedDate()|date(date_format) }}">
     Grades Released Date (grades will be visible to students)
 
-    <br />
-
     <div class="electronic_file">
+        <div class="regrade_request_date">
+            {% if is_regrade_allowed == true and regrade_enabled == true %}
+                <input name="regrade_request_date" id="regrade_request_date" class="date_picker" type="text" value="{{ gradeable.getRegradeRequestDate()|date(date_format) }}">
+                Regrade Request Due Date (Regrade Requests must be sent by this date)
+                <br />
+            {% endif %}
```

Comment:
We always need to have this date input even if regrade requests aren't enabled for this gradeable (it's just display:none)